We live in modern world and have ANSI C here. Plus, nothing defines ANSI_PROTOTYPES
Include unconditionally
diff -ru '--exclude=*.o' streamripper-1.64.6.orig/lib/argv.c streamripper-1.64.6/lib/argv.c
--- streamripper-1.64.6.orig/lib/argv.c	2024-04-03 16:06:09.968107390 +0000
+++ streamripper-1.64.6/lib/argv.c	2024-04-03 16:08:19.399346736 +0000
@@ -52,25 +52,10 @@
 
 /*  Routines imported from standard C runtime libraries. */
 
-#ifdef ANSI_PROTOTYPES
-
 #include <stddef.h>
 #include <string.h>
 #include <stdlib.h>
 
-#else	/* !ANSI_PROTOTYPES */
-
-#if !defined _WIN32 || defined __GNUC__
-extern char *memcpy ();		/* Copy memory region */
-extern int strlen ();		/* Count length of string */
-extern char *malloc ();		/* Standard memory allocater */
-extern char *realloc ();	/* Standard memory reallocator */
-extern void free ();		/* Free malloc'd memory */
-extern char *strdup ();		/* Duplicate a string */
-#endif
-
-#endif	/* ANSI_PROTOTYPES */
-
 
 #ifndef NULL
 #define NULL 0
diff -ru streamripper-1.64.6.orig/lib/ripstream.c streamripper-1.64.6/lib/ripstream.c
--- streamripper-1.64.6.orig/lib/ripstream.c	2024-04-03 16:17:53.035975536 +0000
+++ streamripper-1.64.6/lib/ripstream.c	2024-04-03 16:23:36.372938434 +0000
@@ -714,7 +714,7 @@
     char bigbuf[HEADER_SIZE] = "";
     ID3V2frame id3v2frame;
 #ifndef WIN32
-    __uint32_t framesize = 0;
+    uint32_t framesize = 0;
 #else
     unsigned long int framesize = 0;
 #endif
